schema/vrs.yaml

```diff
@@ -396,27 +387,77 @@ definitions:
     required: [ "type", "gene_id" ]
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Range
+  # Numerics, Comparators, and Ranges
+
+  IntegerOrRange:
```

Comment:
Idea: How about creating a naming convention for internal classes/abstractions that are never instantiated? e.g., _IntegerOrRange.

Note that my preference is to do away with this abstraction entirely, in which case the name suggestion is moot.

Reply:
Okay, will remove.